code/modules/vending/clothingbooth_items.dm

```diff
@@ -616,3 +616,9 @@
 	name = "Green Visor"
 	category = "Accessories"
 	cost = PAY_TRADESMAN/5
+
+/datum/clothingbooth_item/antlers
+	path = /obj/item/clothing/head/antlers
+	amount = 3
+	cost = 200
```

Comment:
Can get rid of this line.

Reply:
Aha, thanks for noticing that! I started working on this before merging in all of the economy changes. My bad.